test/parallel/test-streams-highwatermark.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const stream = require('stream');
+
+const readable = stream.Readable({ highWaterMark: 2147483648 });
```

Comment:
Can you move all of the `2147483648`s into a variable, and add a comment on why that specific value was chosen. That should prevent anyone from changing it to a small value in the future.